src/stream/src/executor/managed_state/aggregation/mod.rs

```diff
@@ -53,6 +54,66 @@ pub fn verify_batch(
     all_lengths.iter().min() == all_lengths.iter().max()
 }
 
+#[derive(Debug)]
+pub struct Cache {
+    synced: bool,            // `false` means not synced with state table (cold start)
+    capacity: Option<usize>, // `None` means unlimited capacity
+    order_pairs: Arc<Vec<OrderPair>>, // order requirements used to sort cached rows
+    rows: BTreeSet<DescOrderedRow>, // in reverse order of `order_pairs`
```

Comment:
Why we use a descending row here?

BTW, we already had a `OrderedRow` before. Can we unify them?